scripts/make_freefield1010_index.py

```diff
@@ -0,0 +1,51 @@
+import argparse
```

Comment:
Different datasets should be added in different PRs to keep better track of things. The best would be to open a different PR and move any `freefield` file there

Reply:
Will make this change and create separate PR for that